src/Microsoft.DocAsCode.Build.ConceptualDocuments/BuildConceptualDocument.cs

```diff
@@ -90,30 +90,13 @@ public override void Build(FileModel model, IHostService host)
             }
         }
 
-        #region ISupportIncrementalBuild Members
+        #region ISupportIncrementalBuildStep Members
 
-        public bool CanIncrementalBuild(FileAndType fileAndType)
-        {
-            return true;
-        }
+        public bool CanIncrementalBuild(FileAndType fileAndType) => true;
 
-        public string GetIncrementalContextHash()
-        {
-            return null;
-        }
+        public string GetIncrementalContextHash() => null;
 
-        public IEnumerable<DependencyType> GetDependencyTypesToRegister()
-        {
-            return new []
-            {
-                new DependencyType
-                {
-                    Name = DependencyTypeName.Include,
-                    IsTransitive = true,
-                    TriggerBuild = true,
-                }
-            };
-        }
+        public IEnumerable<DependencyType> GetDependencyTypesToRegister() => null;
```

Comment:
move to default (at DependencyGraph.cs line 19).

---
In reply to: [79351820](https://github.com/dotnet/docfx/pull/717#discussion_r79351820) [](ancestors = 79351820)